.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+
+on: [push] # without pull_request for the moment
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  specs:
+    name: 'run-specs'
+    # skip on [ci skip] and do not run 2 on push and interal PR
+    if: (contains(github.event.commits[0].message, '[ci skip]') == false) &&  (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
```

Comment:
```suggestion
    if: (contains(github.event.commits[0].message, '[ci skip]') == false) && (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository)
```

Reply:
FYI you probably want `github.event.head_commit.message` 

https://stackoverflow.com/a/68034803/7987708

Reply:
also, I have no idea what `(contains(github.event.head_commit.message, '[ci skip]')` will return for `push`, which doesn't define `head_commit`